internal/command/command.go

```diff
@@ -143,6 +147,25 @@ func (c Command) AddToParent(parent *cobra.Command) {
 	bindFlags(c)
 	parent.AddCommand(c.Cmd)
 }
+func getFullCommand(command *cobra.Command) string {
+	parentCommand := command
+	//var commandSlice []string
+	fullCommand := ""
+
+	for {
+		if parentCommand.Name() == "flow" {
+			return fullCommand
+		} else {
+			//commandSlice = append(commandSlice, parentCommand.Name())
+			if fullCommand != "" {
+				fullCommand = "-" + fullCommand
+			}
+			fullCommand = parentCommand.Name() + fullCommand
+			parentCommand = parentCommand.Parent()
+		}
+	}
+	return ""
+}
```

Comment:
Not needed, just use `command.CommandPath()`

Reply:
addressed